app/view/components/StyledText.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {I18nManager} from 'react-native';
+
+import {useTheme, Text} from 'react-native-paper';
+
+const StyledText = (props) => {
+  const {colors, fonts} = useTheme();
```

Comment:
Why it should calculate in each render? Could you use a `useMemo` hook? I doubt it.

Reply:
@Tohid1999 What's your opinion about this?

Reply:
@amerllica Do you mean to put `useTheme` in `useMemo`?